views/partials/sidebar.ejs

```diff
@@ -1,16 +1,26 @@
-<ul>
-    <li>
-        <img src="https://img.icons8.com/material-rounded/30/000000/home-page.png"/>
-        <a href="/">Home</a>
-    </li>
-    
-    <li>
-        <img src="https://img.icons8.com/material-rounded/30/000000/user-male-circle.png"/>
-        <a href="/profile">Profile</a>
-    </li>
-    <li>
-        <img src="https://img.icons8.com/ios-glyphs/30/000000/gear.png"/>
-        <a href="/settings">Settings</a>
-    </li>
-    <%- include('login') %>
-</ul>
\ No newline at end of file
+<% if (userId === undefined) { %>
+    <ul>
+        <li>
+            <img src="https://img.icons8.com/material-rounded/30/000000/home-page.png"/>
+            <a href="/">Home</a>
+        </li>
+        <%- include('login') %>
+    </ul>
+<% } else { %>
+    <ul>
+        <li>
+            <img src="https://img.icons8.com/material-rounded/30/000000/home-page.png"/>
```

Comment:
This is code duplication. You should probably _always_ have a <ul><li>Home</li></ul> and then put your if block inside of the UL to only display the lis when the user is logged in.

So instead of `if ( userId === undefined ) { ` you could check `if ( userId ) { /* display lis */`